### ISSUE TITLE
Fix Gallery Feature Keyboard & Screen Reader Accessibility

### ISSUE THREAD

**Issue by vkoves** [vkoves/vkoves.github.io#11]
The gallery slideshow feature uses `<img>` elements for buttons, which is not proper semantic HTML and is not accessible via the keyboard. These buttons also lack an accessible name that indicates what they do. Here's a screenshot of the relevant feature, with a button being inspected:

![screenshot from 2018-12-09 15-58-44](https://user-images.githubusercontent.com/3187531/49703483-581c9080-fbcb-11e8-9369-f51e7004fc23.png)

Notice how Chrome does not recognize an accessible name for the element.
